web/styles/modal.css

```diff
@@ -658,6 +658,12 @@
     width: 14em;
 }
 
+.email_change_container .settings-field-hint {
+    /* Constrain warning text width to match input field area
+       (input: 14em + suffix text) for proper wrapping. */
+    max-width: 25em;
```

Comment:
I think it'll probably look better if we actually let this get wider, so that it's one line.